api/go/gen/pub/sandbox/v1alpha1/sandboxv1alpha1connect/sandbox.connect.go

```diff
@@ -11,7 +11,7 @@ import (
 	connect "connectrpc.com/connect"
 	context "context"
 	errors "errors"
-	v1alpha1 "go.jetpack.io/axiom/api/gen/pub/sandbox/v1alpha1"
+	v1alpha1 "go.jetify.com/axiom/api/gen/pub/sandbox/v1alpha1"
```

Comment:
was this auto-gen'd or manually edited? :)

Reply:
I did a search and replace across all the files (so it replaced inside the generated files). Once I have the first change in, I'll regenerate files as well.